pkg/pillar/hypervisor/kvm.go

```diff
@@ -527,12 +527,20 @@ func (ctx kvmContext) CreateDomConfig(domainName string, config types.DomainConf
 		PCIId, NetID     int
 		Driver           string
 		Mac, Bridge, Vif string
+		Vlan             string
```

Comment:
You add this for KVM. Is there also some addition needed for Xen?

Reply:
I should have removed this. This is a remnant from my tries to make the vlan application through hypervisors. Will remove this code.